.gitignore

```diff
@@ -0,0 +1,15 @@
+# dependencies
+node_modules
+
+# production
+dist
+
+# misc
+.DS_Store
+
```

Comment:
Could you add a `.env` and a `env.list`, please ? 🙏

Reply:
Definitely, done with https://github.com/Major-Eth/yearn-web-lib/pull/8/commits/bbf7452976845a1d860eb13220782111b5cedf7b